### ISSUE TITLE
Issue: Number tests may fail (with -Duselongdouble?)

### ISSUE THREAD

**Issue by eserte** [iamalnewkirk/data-object#160]
On two of my smokers the test suite fails:
```
#   Failed test at t/0.90/can/Data/Object/Func/Number/Atan2/execute.t line 57.
#          got: '0.78539816339744831'
#     expected: '0.785398163397448'
# Looks like you failed 1 test of 3.
t/0.90/can/Data/Object/Func/Number/Atan2/execute.t .......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
...
#   Failed test at t/0.90/can/Data/Object/Func/Number/Cos/execute.t line 55.
#          got: '0.843853958732492105'
#     expected: '0.843853958732492'
# Looks like you failed 1 test of 3.
t/0.90/can/Data/Object/Func/Number/Cos/execute.t ............ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
... (etc) ...
```

This seems to happen if perl is compiled with long doubles (`-Duselongdouble`).

**Comment by eserte** [comment]
A sample fail report from a different smoke system: http://www.cpantesters.org/cpan/report/492a1f40-46ab-11e9-bd19-95c55aa544d1

**Comment by iamalnewkirk** [comment]
Thanks for the report. I've addressed the issue which will be applied with the next release.

**Comment by iamalnewkirk** [comment]
Fixed in #166
